src/test/java/spoon/reflect/factory/PackageFactoryTest.java

```diff
@@ -0,0 +1,32 @@
+package spoon.reflect.factory;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.sameInstance;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import spoon.Launcher;
+import spoon.reflect.declaration.CtPackage;
+
+class PackageFactoryTest {
+
+	@Test
```

Comment:
Could you add either a comment or the GitHub issue linking to the issue of the bug? This could help to understand test cases in the future.

Reply:
Good catch, I routinely forget that we have that new annotation. Fixed.